miden-wasm/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+use miden::{Assembler, ProgramInputs, ProofOptions};
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_test::*;
+extern crate console_error_panic_hook;
+
+
+/// We derive Deserialize/Serialize so we can persist app state on shutdown.
+#[derive(serde::Deserialize, serde::Serialize)]
+pub struct InputFile {
+    pub stack_init: Vec<String>,
+    pub advice_tape: Option<Vec<String>>,
+}
+
+fn get_program_inputs(stack_init: &[u64], advice_tape: &[u64]) -> ProgramInputs {
+    ProgramInputs::new(&stack_init, &advice_tape, Vec::new()).unwrap()
+}
+
+/// Parse stack_init vector of strings to a vector of u64
+fn stack_init(inputs_des: &InputFile) -> Vec<u64> {
```

Comment:
minor nit (feel free to ignore): I think using `input` instead of `inputs_des` might be better.